registry/server/migrations/20210426163602_routes_orderPosIsNullable.ts

```diff
@@ -0,0 +1,84 @@
+import * as Knex from "knex";
+
+export async function up(knex: Knex): Promise<any> {
+    if (isMySQL(knex)) {
+        return knex.schema.alterTable('routes', table => {
+            table.integer('orderPos', 10).nullable().alter(); // alter table don't affect only uniq constraint, so you don't need and it's disallowed to set unique('routes_orderpos_unique') here
+            table.string('route', 255).nullable().alter();
+            table.boolean('next').nullable().alter();
```

Comment:
Why would you make it null? Just fallback to false

Reply:
reverted